# Generated by Django 2.2.3 on 2019-08-24 19:17

from django.conf import settings
from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.CreateModel(
            name='ModelCountRequests',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('counter', models.IntegerField()),
                ('updated_at', models.DateTimeField(auto_now_add=True)),
            ],
        ),
        migrations.CreateModel(
            name='ModelRequestByUser',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('created_at', models.DateTimeField(auto_now_add=True)),
                ('input_cidade', models.CharField(max_length=250)),
                ('input_bairro', models.CharField(max_length=250)),
                ('input_rua', models.CharField(blank=True, max_length=250, null=True)),
                ('input_numero', models.CharField(blank=True, max_length=250, null=True)),
                ('input_estado', models.CharField(max_length=250)),
                ('confidence', models.CharField(blank=True, max_length=250, null=True)),
                ('search_address', models.CharField(blank=True, max_length=250, null=True)),
                ('geocode_lat', models.FloatField(blank=True, null=True)),
                ('geocode_lng', models.FloatField(blank=True, null=True)),
                ('errors', models.TextField(blank=True, null=True)),
                ('user', models.ForeignKey(on_delete=models.SET('Undefined User'), to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
